lib/lte_link_control/modules/ncellmeas.c

```diff
@@ -667,6 +667,13 @@ static void at_handler_ncellmeas(const char *response)
 		goto exit;
 	}
 
+	if (k_sem_count_get(&ncellmeas_idle_sem) > 0) {
+		/* No need to parse the notification because neighbor cell measurement
+		 * has not been requested.
+		 */
```

Comment:
I'm thinking if we should have a log here. Not ERR or WRN but at least DBG if not INF.
`LOG_DBG("%%NCELLMEAS notification not parsed because lte_lc_neighbor_cell_measurement() not called");`

Reply:
This is not as straight forward as we first thought. There's a problem related to `lte_lc_neighbor_cell_measurement_cancel()`. When neighbor cell measurement is cancelled, there can still be a `%NCELLMEAS` notification from the modem which should be parsed. There's no way to know if there will be a notification or not, because it depends on whether the modem had started the measurement or not. This needs some more thought, I marked the PR as DNM.